### DISCUSSION TITLE
Missing Dashboard Graph and Compilation

### DISCUSSION THREAD

**Discussion by Jattura** [OpenMage/magento-lts#3212] [Questions & Answers] [answered]
### Preconditions (*)

1. 19.5.0-rc2
2.

### Steps to reproduce (*)

Hello, Team 

I suddenly found two problems. It is blank. 

1. Dashboard Amounts / Orders without graph 

2. Compilation

The way I installed it was by copying the new version file and pasting in the old one. 

Please advise how to solve it. 

Thank you! 

### Expected result (*)

1. 
![圖片](https://user-images.githubusercontent.com/127373501/226914621-145f1836-7f26-4a83-815c-419dfdb60736.png)

2. 
![圖片](https://user-images.githubusercontent.com/127373501/226914655-bd086094-d47b-4db5-9dbc-9c01a8f0c4d8.png)

### Actual result (*)

1. Blank
2.

**Comment by Jattura** [comment]
It's a compilation error record

![圖片](https://user-images.githubusercontent.com/127373501/227407614-3be6014a-75cb-4d3c-93ae-f897fe29dfd8.png)

**Comment by elidrissidev** [comment]
Compiler module was removed ages ago alongside other modules, consult the README for the full list.

**Comment by addison74** [comment]
Indeed Compiler feature was removed from OpenMage long time ago. 

Regarding the lack of the chart block in the Dashboard, it was a known issue in the Magento source code, but it was solved both in Magento and in OpenMage. I recommend you to look through the logs of OpenMage, PHP, the browser console.

**Comment by Jattura** [comment]
Thank you for your reply. Yesterday, my sister wanted to check her previous customer order records. When she clicks to view the order page, the page is blank. Then, I look at other previous orders, some I can see, some I don't. It is very strange, click the order menu, the order list is still there, but the above situation occurs when you click to view.

I can't see any logs and reports. So, I guess that's why the graph disappears. I think RC3 will work after placing the file where I can see the graph. But this test still doesn't work.  I then copied the previous php7.4 version into the sales and admin sales folders. It didn't work either.

Elidrissidev is aware of my limited knowledge background. Perhaps, please let me know where is wrong with the file, so, I will check. Thanks.

**Comment by addison74** [comment]
Unfortunately, I cannot reproduce the problem reported in a test version of OpenMage with Magento Sample Pack (with PHP 7.4 - 8.2). I opened random orders and all contain information. You need technical support and I recommend to find a knowledgeable person to help you identify the problem. 

I will move this report to the Discussions > Q&A section.

**Comment by elidrissidev** [comment, marked as answer]
I did confirm this issue in their store, the chart wasn't loading even though the data that was sent to charts api was correct. Unfortunately, i'm just too low on time and motivation currently to dig into the issue.
